### PR TITLE
Update MILLISECS_PER_BLOCK comments

### PR DESCRIPTION
**Pull request by apopiak**
+ Adds a doc comment to the node template version of `MILLISECS_PER_BLOCK`.
+ Fixes broken Babe block production link

### CODE REVIEW

**Comment by apopiak** [comment]
related: https://github.com/w3f/research/issues/105

File: bin/node/runtime/src/constants.rs [hunk 1]
```diff
@@ -50,7 +50,7 @@ pub mod time {
 	/// always be assigned, in which case `MILLISECS_PER_BLOCK` and
 	/// `SLOT_DURATION` should have the same value.
 	///
-	/// <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
+	/// This link is broken: <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
```
Comment (andresilva) [inline bin/node/runtime/src/constants.rs:53]:
```suggestion
	/// <https://research.web3.foundation/en/latest/polkadot/block-production/Babe.html#-6.-practical-results>
```

**Review by andresilva — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by apopiak** [comment]
bot merge

**Comment by apopiak** [comment]
mmh, getting an ICE in the CI?

**Pull request merged**